src/ViewModels/test/RemoteDebugViewModelTests.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Tanzu.Toolkit.ViewModels.RemoteDebug;
+
+namespace Tanzu.Toolkit.ViewModels.Tests
+{
+    [TestClass]
+    public class RemoteDebugViewModelTests : ViewModelTestSupport
+    {
+        private const string _fakeTargetFrameworkMoniker = "some fake tfm";
+        private const string _fakePathToLaunchFile = "fake\\path\\to\\launch\\file";
+        private RemoteDebugViewModel _sut;
+
+        [TestInitialize]
+        public void TestInit()
+        {
+            MockTasExplorerViewModel.SetupGet(m => m.TasConnection).Returns((CfInstanceViewModel)null);
+            MockThreadingService.Setup(m => m.StartBackgroundTask(It.IsAny<Func<Task>>(), It.IsAny<CancellationToken>())).Verifiable();
+
+            _sut = new RemoteDebugViewModel(_fakeAppName, _fakeProjectPath, _fakeTargetFrameworkMoniker, _fakePathToLaunchFile, null, Services);
+        }
+
+        [TestCleanup]
```

Comment:
For future use? Doesn't matter much to me, but if there are already test cleanup examples in the solution it may not be worth having placeholders like this

Reply:
Done in [00f640a](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/pull/354/commits/00f640a66b3a73deab813f08d13fa266c6be7afc)